developer-docs-site/docs/guides/state-sync.md

```diff
@@ -24,18 +24,20 @@ State sync runs in two modes. All nodes will first bootstrap (in bootstrapping m
 
 ### Bootstrapping mode
 
-When the node starts, state sync will perform bootstrapping by using the specified bootstrapping mode configuration. This allows the node to catch up to the Aptos blockchain. There are three bootstrapping modes:
+When the node starts, state sync will perform bootstrapping by using the specified bootstrapping mode configuration. This allows the node to catch up to the Aptos blockchain. There are several bootstrapping modes:
 
 - **Execute all the transactions since genesis**. In this state sync mode the node will retrieve from the Aptos network all the transactions since genesis, i.e., since the start of the blockchain's history, and re-execute those transactions. Naturally, this synchronization mode takes the longest amount of time.
 - **Apply transaction outputs since genesis**. In this state sync mode the node will retrieve all the transactions since genesis but it will skip the transaction execution and will only apply the outputs of the transactions that were previously produced by validator execution. This mode reduces the amount of CPU time required.
+- **(Default) Execute or apply since genesis**. In this state sync mode the node will retrieve all the transactions since genesis and will either execute the transactions, or apply the transaction outputs, depending on whichever is faster, per data chunk. This allows the node to adapt to CPU and network resource constraints more efficiently. This mode is the default mode.
```

Comment:
For consistency, could we always mention "intelligent" for the syncing mode? I think it's a bit confusing whether they're the same thing or not.

Reply:
Good point. I've updated the wording. I've also fixed any inconsistent wording for fast sync, too 😄